scripts/update

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GEOSITE_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n "Usage: $(basename "$0")
+    Attempts to install dependencies.
```

Comment:
We may have considered this, but would it make sense to have these installed via Dockerfile?

Reply:
Nevermind, I see that this is done in https://github.com/CoastalResilienceNetwork/GeositeFramework/pull/1111/commits/e3af147593d39e6cdeb49e6841ddc5cb6cfb259a